bacs/src/main/java/com/adyen/checkout/bacs/internal/ui/DefaultBacsDirectDebitDelegate.kt

```diff
@@ -121,6 +125,9 @@ internal class DefaultBacsDirectDebitDelegate(
     }
 
     override fun onSubmit() {
+        val event = GenericEvents.submit(paymentMethod.type.orEmpty())
```

Comment:
This should probably be moved in the `BacsDirectDebitMode.CONFIRMATION` case of the `when` down below